### PR TITLE
Update i18next dependency to 2.x

### PR DESCRIPTION
**Pull request by fodor0205**

### CODE REVIEW

**Comment by ypid** [comment]
That is solid work, thank you very much!

**Comment by ypid** [comment]
Is there a specific reason you pinned on i18next ^2.5.1? I propose to upgrade to the latest version in #386.

**Comment by fodor0205** [comment]
No specific reason, just to make this PR as small as possible and the initial work a little easier. (By excluding any possible breaking changes between v2.x and v19.x in this first round).

Now I checked it again:
I don't see any risk in updating to the latest version, the tests are quite good in this project and `i18next` also has good [changelog](https://github.com/i18next/i18next/blob/master/CHANGELOG.md) since 2.0.

**Comment by ypid** [comment]
Sounds good. So I am merging #386.

File: site/index.html [hunk 1]
```diff
@@ -10,8 +10,9 @@
         <link rel="stylesheet" href="css/table.css"/>
         <!-- Javascript {{{ -->
         <script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha384-rY/jv8mMhqDabXSo+UCggqKtdmBfd3qC2/KvyTDNQ6PcUJXaxK1tMepoQda4g5vB" crossorigin="anonymous"></script>
-        <script src="opening_hours+deps.min.js"></script>
-        <script src="node_modules/i18next-client/i18next.min.js"></script>
+        <script src="../build/opening_hours+deps.min.js"></script>
+        <script src="../node_modules/i18next/dist/umd/i18next.min.js"></script>
+        <script src="https://unpkg.com/i18next-browser-languagedetector/i18nextBrowserLanguageDetector.min.js"></script>
```
Comment (ypid) [inline site/index.html:15]: I just saw that you included an external resource without even using https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity. I fixed that in #391

**Pull request merged**